aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -2075,16 +2075,13 @@ Tensor matrix_exp(const Tensor& a) {
   return at::linalg_matrix_exp(a);
 }
 
-Tensor linalg_matrix_exp_backward(const Tensor& self, const Tensor& grad) {
+Tensor linalg_matrix_exp_backward(const Tensor& self, const Tensor& grad, bool backward_ad) {
```

Comment:
Why is this function implemented here and not in FunctionsManual.cpp ? Should we move it?

Reply:
That's a good question. I left it there because it was there. What is the difference between a function being here or in `FunctionsManual.cpp`?

Reply:
I think the difference is whether it's available in the `at` namespace or not.

Reply:
I think we should move it then. @albanD what do you think?

Reply:
Yes availability in the at namespace, and if you need to define a backward for it, you need to have it there.

The other case is if the implementation is not generic enough (for batched Tensors or other backends) and we need to be able to overwrite it for them, then it needs to be in native_functions as well.

But if nothing breaks when you move it back, you can definitely do it.